app/profile/[id]/manageTask/page.tsx

```diff
@@ -33,32 +36,46 @@ const FULL_TASK_LVL_IMPORTANCE_LIST: ExtendedImportanceLevelTaskType[] = [
 export default function ManageTask() {
   const [tasksData, setTasksData] = useState<DataType[] | []>([]);
   const [currentCategory, setCurrentCategory] = useState(
-    ExtendedCategoryTaskType.ALL
+    ExtendedCategoryTaskType.DEFAULT
+  );
+  const [currentImportance, setCurrentImportance] = useState(
+    ExtendedImportanceLevelTaskType.DEFAULT
   );
-  const [currentImportance, setCurrentImportance] = useState("all");
   const [currentTasksData, setCurrentTasksData] = useState<DataType[]>([]);
   const userId = useAppSelector(getUserId);
 
   const getFilteredTasks = () => {
-    if (
-      currentCategory === ExtendedCategoryTaskType.ALL &&
-      currentImportance === ExtendedImportanceLevelTaskType.ALL
-    ) {
+    const showAllCategory =
+      currentCategory === ExtendedCategoryTaskType.ALL ||
+      currentCategory === ExtendedCategoryTaskType.DEFAULT;
+
+    const showAllImportanceLevel =
+      currentImportance === ExtendedImportanceLevelTaskType.ALL ||
+      currentImportance === ExtendedImportanceLevelTaskType.DEFAULT;
+    if (showAllCategory && showAllImportanceLevel) {
       setCurrentTasksData(tasksData);
     } else {
       const categoryFilteredTasks = tasksData.filter((task: DataType) => {
         const categoryUpperCase = task.task.category.toUpperCase();

```

Comment:
Trochę tutaj to zagnieżdżenie lekko urwiera w oczy, że jest task.task, jeżeli da się to prosto fixnąć to możesz spróbować, jeśli nie to świat się nie zawali.